lib/evss/disability_compensation_form/data_translation_all_claim.rb

```diff
@@ -387,7 +387,7 @@ def translate_treatments
             'endDate' => approximate_date(treatment['treatmentDateRange']['to']),
             'treatedDisabilityNames' => treatment['treatedDisabilityNames'],
             'center' => {
-              'name' => treatment['treatmentCenterName'].gsub(/[^a-zA-Z0-9 .()#&'"-]+/, '').strip
+              'name' => treatment['treatmentCenterName'].gsub(/[^a-zA-Z0-9 .()#&'"-]+/, '').gsub(/\s\s+/, ' ').strip
```

Comment:
Maybe this is stupid to do two gsubs, but I was worried about how difficult it would be to read a combined statement.

Reply:
it's necessary to do in 2.  Otherwise, space-comma-space would not become a single space.